controlplane/rosa/api/v1beta2/rosacontrolplane_types.go

```diff
@@ -99,6 +99,26 @@ type RosaControlPlaneSpec struct { //nolint: maligned
 	// Autoscaling specifies auto scaling behaviour for the MachinePools.
 	// +optional
 	Autoscaling *expinfrav1.RosaMachinePoolAutoScaling `json:"autoscaling,omitempty"`
+
+	// +kubebuilder:validation:Optional
+
+	// AdditionalTags are user-defined tags to be added on the AWS resources associated with the control plane.
+	AdditionalTags infrav1.Tags `json:"additionalTags,omitempty"`
```

Comment:
since you removed the webhook, we need to add the validation for this in runtime inside `validateControlPlaneSpec()` func.
 https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/4e748e20799c96f20300aeedef461a1608220b2e/api/v1beta2/tags.go#L77

Reply:
Good catch - not in the previous PR even with the webhook, but let me add.